email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -1401,6 +1408,31 @@ const handleAddressCreatedEvent = async ({ rowid, params }, accountId) => {
   return { rowid };
 };
 
+const handlePeerSetTrustedEvent = async ({ rowid, params }) => {
+  const { email, trusted } = params;
+  await updateContactByEmail({ email, isTrusted: trusted });
+  const [contact] = await getContactByEmails({ emails: [email] });
+  if (contact) {
+    const contactId = contact.id;
+    emitter.emit(Event.CHANGE_SET_TRUSTED_ACCOUNT, {
```

Comment:
Look at `handlePeerLabelUpdate` to update multiple contacts as a batch instead of going one by one during event parse